### DISCUSSION TITLE
When I start a mod in a bottles prefix but the base game isn't located in the bottles prefix,will it still work then?

### DISCUSSION THREAD

**Discussion by ghost** [bottlesdevs/Bottles#357] [Q&A]
So what I want to do is to launch a mod for a steam game.
The mod itself is in the prefix in which the game is located.
I want to open that mod with a bottles prefix that has the same proton version as the proton prefix in which the mod is.
My question is,if I open that mod in that bottles prefix,would it work or would I have to open the mod in the steam prefix? (if I have to open it in the steam prefix,how can I do that?)

**Comment by mirkobrombin** [comment]
I don't know, it seems more like a question for the makers of Proton or Proton GE if you use that variant. 

I guess you have to go through Proton but I can't tell you how.

**Reply by ghost** [reply]
Then what if the base game and the mod is located on my linux file system and I open it with a bottle,will it then work or would I have to move the base game and the mod into the bottle?
